civihr_employee_portal/css/custom.css

```diff
@@ -281,7 +285,7 @@ span.appraisals-employee-legend {
   padding: 8px 20px 8px 15px;
   position: relative;
   white-space: normal;
-  width: inherit;
+  width: 100%;
```

Comment:
Why is this necessary?

Reply:
@bmnepali suggested using `width: inherit` instead of `width: 100%` and that worked well for one of the suggestions, but I didn't notice it didn't work for the fields in the row area:

**Before:**
![people reports hr17 9000 2](https://user-images.githubusercontent.com/1642119/34670193-a26e3752-f44b-11e7-8a31-fb9b35e9a096.png)

**After:**
![people reports hr17 9000 1](https://user-images.githubusercontent.com/1642119/34670164-84e2a7cc-f44b-11e7-94a1-747a3702e4fc.png)